<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>509</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>ATROX</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LPSolve</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/LPSolve.png.icns</normaloff>:/LPSolve.png.icns</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(99, 99, 99);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

#solveButton {
	border: 0px solid rgb(239, 239, 239);
	background-color: rgb(144, 145, 145);
	border-radius: 10px;	
	color: rgb(0, 0, 0);
}

#solveButton:hover
{
	border: 0px solid rgb(252, 252, 252);
	background-color: rgb(207, 209, 209);
	border-radius: 10px;
	color: rgb(0,0,0);
}

#solveButton:pressed
{
	/*background-color: rgb(208, 20, 33, 220);*/
	border: 0px solid rgb(252, 252, 252);
	background-color: rgb(179, 179, 179, 10);
	border-radius: 10px;
color: rgb(208, 20, 33, 220);
}</string>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="maxRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-1</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#maxRadioButton:checked {
	selection-background-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="minRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="objFuncBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(207, 209, 209);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>objective Function:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="constraintsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(207, 209, 209);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Constraints:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>351</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(207, 209, 209);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>output:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="font">
    <font>
     <family>Copperplate</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
   <property name="font">
    <font>
     <family>Copperplate</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
